ghinsights/USQL/ProcessDaily.usql

```diff
@@ -1967,6 +1967,44 @@ WHERE RowNumber == 1;
 //  Process PullRequestComment
```

Comment:
This comment goes with the code below for PullRequestComment

Can you add PullRequestCommit so it doesn't split the comment from the code?